zerver/views/users.py

```diff
@@ -373,16 +382,16 @@ def get_member(row):
 
         result['is_admin'] = user_id in admin_ids
 
-        avatar_user_dict = dict(
-            id=user_id,
+        result['avatar_url'] = get_avatar_field(
+            user_id=user_id,
+            email=email,
             avatar_source=row['avatar_source'],
             avatar_version=row['avatar_version'],
-            email=email,
             realm_id=row['realm_id'],
+            medium=False,
+            client_gravatar=client_gravatar,
         )
```

Comment:
@borisyankov is your expectation for this option that there would just not be an `avatar_url` field in the response, or that it would be `None`?  I guess with compression those are pretty similar in terms of the data size, but I believe this implementation would pass `None` rather than skipping the field.

Reply:
An empty field looks more conventional, and indeed it does not make much difference in file size.

Reply:
I'm actually not sure which of the two options you mean with "an empty field"